data/custom-formats.yaml

```diff
@@ -50,4 +50,7 @@ mx-mxc-uri:
 uri:
   title: URI
   url: http://tools.ietf.org/html/rfc3986
```

Comment:
While we're here, could we switch this to an HTTPS url please?

Reply:
This URL was updated in #1977.